packages/fetchai/skills/coin_price/handlers.py

```diff
@@ -204,3 +219,71 @@ def teardown(self) -> None:
         :return: None
         """
         pass
+
+
+class PrometheusHandler(Handler):
+    """This class handles responses from the prometheus server."""
+
+    SUPPORTED_PROTOCOL = PrometheusMessage.protocol_id
+
+    def __init__(self, **kwargs):
+        """Initialize the handler."""
+        super().__init__(**kwargs)
+
+        self.handled_message = None
+
+    def setup(self) -> None:
+        """Set up the handler."""
+        if self.context.prometheus_dialogues.enabled:
+            self.context.logger.info("setting up PrometheusHandler")
```

Comment:
Yes, this is true. I partly wanted to avoid doing a conditional import like we did for the `http_server`. It's a bit messy to give agents optional use of packages. Maybe something to keep in mind for the future, but for now we can just always log this setup info.